cmd/kk/apis/kubekey/v1alpha2/default.go

```diff
@@ -90,6 +90,7 @@ const (
 	DefaultDNSAddress           = "114.114.114.114"
 	DefaultDpdkTunnelIface      = "br-phy"
 	DefaultCNIConfigPriority    = "01"
+	DefaultOpenEBSBasePath      = "/var/openebs"
```

Comment:
Suggest `/var/openebs/local/`, because that is the same path as in the previous code.